cmake/detect-sanitizer.cmake

```diff
@@ -153,12 +150,6 @@ macro(add_undefined_sanitizer)
         add_compile_options(-fsanitize=${supported_checks})
         add_link_options(-fsanitize=${supported_checks})
 
-        # Group sanitizer flag -fsanitize=undefined will automatically add alignment, even if
-        # it is not in our sanitize flag list, so we need to explicitly disable alignment sanitizing.
-        if(WITH_UNALIGNED)
-            add_compile_options(-fno-sanitize=alignment)
-        endif()
-
```

Comment:
Awesome, hopefully by removing this bit we'll have some coverage for anything we've missed with CI.

Reply:
Yes, looks like we already found an invalid alignment access at crc32_acle.c:52